rapids-cmake/cmake/support_conda_env.cmake

```
@@ -76,12 +76,49 @@ function(rapids_cmake_support_conda_env target)
 
   if(in_conda_build OR in_conda_prefix)
 
+    #
+    macro(modify_cmake_prefix_path_global)
+      cmake_parse_arguments(_RAPIDS "" "" "PATHS" ${ARGN})
+
+      if(NOT ("$ENV{CMAKE_PREFIX_PATH}" STREQUAL ""))
```

```suggestion
      if(DEFINED ENV{CMAKE_PREFIX_PATH})
```
I think that makes it clearer